### ISSUE TITLE
Reflective rain ground still not fixed insidew

### ISSUE THREAD

**Issue by churchner420** [BruceKnowsHow/Ebin-Shaders#23]
Nvidia drivers: 381.89
Minecraft version: 1.11.2
Mods: Optifine (only, installed through the installer)
Computer specs: GTX950M, 8gb DDR4 2133mhz, Intel Core i5-6200U
Issue: Although it says fixed in your bug report the bug with the reflective ground inside when raining is still there.
![2017-05-10_19 05 23](https://cloud.githubusercontent.com/assets/18448228/25886959/200c8706-35b4-11e7-86ac-d348aea2d37a.png)

**Comment by soraphis** [comment]
had the same problem, was fixed for me after i disabled the weather option in the shader again

**Comment by churchner420** [comment]
Yeah I know... 😒 but it disables all reflections including the outdoors as well.
